CatchMe/CatchMe/Source/ViewControllers/CharacterVC.swift

```diff
@@ -127,9 +185,13 @@ extension CharacterVC: UITableViewDataSource {
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         switch section {
         case 0:
-            return 1 + 10
+            if posts.isEmpty == true {
```

Comment:
```suggestion
            if posts.isEmpty {
```
== true 떼도 충분히 true라는 의미라서 괜찮습니다!